lumigator_demo.py

```diff
@@ -90,6 +91,24 @@ def get_job_status(response: requests.Response) -> UUID:
         return json.loads(response.text).get("status")
 
 
+def download_text_file(response: requests.Response) -> str:
```

Comment:
suggestion `download_file` so we don't have to specify the type of file later on

Reply:
I wanted to be explicit on this because ATM we cannot download e.g. HF datasets this way, but just individual file. I will rename this and make sure its limitation is clear in the comments